sdk/storage/azure-storage-blob/azure/storage/blob/blob_client.py

```diff
@@ -221,6 +222,7 @@ def from_connection_string(
         return cls(
             account_url, container=container, blob=blob, snapshot=snapshot, credential=credential, **kwargs)
 
+    @distributed_trace
```

Comment:
I don't think this function does a network call.

Reply:
ok decorator is no longer there